project1-sps/script.js

```diff
@@ -1,64 +1,54 @@
-var numberOfGames =0; 
-
+var win = 0;
+var draw = 0;
+var lose = 0;
+var numberOfGames = 0;
 
 var main = function (input) {
-  var computerGuess = rollDice(); 
+  var computerGuess = rollDice();
 
-  if (computerGuess !=0){
-    numberOfGames +=1;
-    var winPercent = Math.floor(win/numberOfGames *100); 
-    var drawPercent = Math.floor(draw/numberOfGames *100); 
-    var LosePercent = Math.floor(lose/numberOfGames *100); 
+  if (computerGuess != 0) {
+    numberOfGames += 1;
+    var winPercent = Math.floor((win / numberOfGames) * 100);
+    var drawPercent = Math.floor((draw / numberOfGames) * 100);
+    var LosePercent = Math.floor((lose / numberOfGames) * 100);
     console.log(`The number of games played is ${numberOfGames}`);
   }
 
-  if (computerGuess == 1){
+  if (computerGuess == 1) {
     output = `scissors`;
 
-    if (input == output){
-      draw +=1; 
+    if (input == output) {
+      draw += 1;
       result = `The computer chose ${output}. You chose ${input}. The result is: Draw. The draw percentage is ${drawPercent} %`;
-    }
-    else if (input == 'paper'){
-      lose +=1;
+    } else if (input == "paper") {
+      lose += 1;
       result = ` The computer chose ${output}. You chose ${input}. The result is: Lose. The lose percentage is ${LosePercent} %`;
-    }
-    else{
-      win +=1;
+    } else {
+      win += 1;
       result = ` The computer chose ${output}. You chose ${input}. The result is: Win. The win percentage is ${winPercent} %`;
     }
-  }
-  else if (computerGuess == 2){
-
+  } else if (computerGuess == 2) {
     output = `paper`;
-    if (input == output){
-      draw +=1; 
+    if (input == output) {
+      draw += 1;
       result = `The computer chose ${output}. You chose ${input}. The result is: Draw. The draw percentage is ${drawPercent} %`;
-    }
-    else if (input == 'scissors'){
-      win +=1;
+    } else if (input == "scissors") {
+      win += 1;
       result = `The computer chose ${output}. You chose ${input}. The result is: Win. The win percentage is ${winPercent} %`;
-    }
-    else{
-      lose +=1;
+    } else {
+      lose += 1;
       result = `The computer chose ${output}. You chose ${input}. The result is: Lose. The lose percentage is ${LosePercent} %`;
     }
-  }
-  else{
+  } else {
     output = `stone`;
-    if (input == output){
-      draw +=1; 
+    if (input == output) {
+      draw += 1;
       result = `The computer chose ${output}. You chose ${input}. The result is: Draw. The draw percentage is ${drawPercent} % `;
-    }
-    else if (input == 'paper'){
-      win = win+1;
+    } else if (input == "paper") {
+      win = win + 1;
       result = `The computer chose ${output}. You chose ${input}. The result is: Win. The win percentage is ${winPercent} %`;
-    }
-    else{
-      lose +=1;
+    } else {
+      lose += 1;
       result = `The computer chose ${output}. You chose ${input}. The result is: Lose. The lose percentage is ${LosePercent} %`;
     }
   }
```

Comment:
The logic is correct but there are a few points I would like to point out to make it more readable

your win , lose , draw messages are the same, maybe you can write a function for them so you dont have to repeat code like this.

Maybe you can try to make the logic a little more concise, what conditions will lead to a win, lose or draw and group them together? Verbose code that works is good but shorter code that works is easier to read!